usaspending_api/search/v2/views/spending_by_category_views/spending_by_recipient.py

```diff
@@ -94,3 +95,15 @@ def query_django_for_subawards(self, base_queryset: QuerySet) -> List[dict]:
                 del row[key]
 
         return query_results
+
+
+@method_decorator(deprecated, name="post")
+class RecipientDunsViewSet(RecipientViewSet):
+    """
+    <em>Deprecated: Please see <a href="recipient">this endpoint</a> instead.</em>
```

Comment:
This links to `GET /api/v2/search/spending_by_category/recipient_duns/recipient`. Shouldn't it actually link to `GET /api/v2/search/spending_by_category/recipient`

Reply:
It does still "work" because of the URL's lack of a end regex character that you mentioned above, but `GET /api/v2/search/spending_by_category/recipient_duns/recipient` isn't an actual endpoint

Reply:
Thank you, great catch.